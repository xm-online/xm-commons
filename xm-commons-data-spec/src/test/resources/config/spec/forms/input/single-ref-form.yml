test-items:
    - key: team/CREATE_EMPLOYEE
      inputForm: |
          {
            "form": [
              {
                "$ref": "#/xmForm/employeeForm"
              }
            ],
            "prop": {
              "hideNameAndDescription": true
            }
          }

forms:
    - key: employeeForm
      value: |-
          [
              { "key": "firstName", "title": { "en": "First name" }, "type": "string" },
              { "key": "lastName", "title": { "en": "Last name" }, "type": "string" },
              { "key": "email", "title": {"en": "Email" }, "type": "string" }
          ]
