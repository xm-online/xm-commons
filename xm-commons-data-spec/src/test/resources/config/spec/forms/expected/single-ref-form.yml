test-items:
    - key: team/CREATE_EMPLOYEE
      inputForm: |
          {
            "form":[
              { "key": "firstName", "title": { "en": "First name" }, "type": "string" },
              { "key": "lastName", "title": { "en": "Last name" }, "type": "string" },
              { "key": "email", "title": {"en": "Email" }, "type": "string" }
            ],
            "prop":{
                "hideNameAndDescription":true
            }
          }
forms:
    - key: employeeForm
      value: |-
          [
              { "key": "firstName", "title": { "en": "First name" }, "type": "string" },
              { "key": "lastName", "title": { "en": "Last name" }, "type": "string" },
              { "key": "email", "title": {"en": "Email" }, "type": "string" }
          ]
